libyul/optimiser/LoadResolver.cpp

```diff
@@ -76,3 +105,62 @@ void LoadResolver::tryResolve(
 			if (inScope(*value))
 				_e = Identifier{locationOf(_e), *value};
 }
+
+void LoadResolver::tryEvaluateKeccak(
+	Expression& _e,
+	std::vector<Expression> const& _arguments
+)
+{
+	// Runtime saving: [PUSH32 32-byte-value] instead of [PUSH1 0 PUSH1 32 KECCAK256]
+	// Saves one KECCAK256 + one PUSH. Currently it is 30 + 6 + 3 = 39
+	unsigned runTimeGasSavings = GasCosts::keccak256Gas + GasCosts::keccak256WordGas + GasCosts::tier2Gas;
+	// Deploy time increase
+	// [PUSH32 32-byte-value]: 33 bytes
+	// instead of
+	// [PUSH1 0 PUSH1 32 KECCAK256]: 5 bytes
+	// Increase of 28 bytes.
+	// Current increase: 28 * 200: 56000
```

Comment:
```suggestion
	// Current increase: 28 * 200 = 56000
```

Reply:
Ah. There is also a typo. It should be `5600`. :(